crates/analyzer/tests/analysis.rs

```diff
@@ -415,8 +415,6 @@ fn function_diagnostics(fun: items::FunctionId, db: &dyn AnalyzerDb) -> Vec<Diag
     [
         // signature
         build_debug_diagnostics(&[(fun.data(db).ast.span, &fun.signature(db))]),
-        // declarations
-        label_in_non_overlapping_groups(&lookup_spans(&body.var_decl_types, &body.spans)),
```

Comment:
How about keeping this line? 
It may be helpful when a type inference is implemented.
```
let my_array: Array<u256, 10>
    ^^^^^^^^ Array<u256, 10>
```